tests/test_fs.py

```diff
@@ -149,3 +171,24 @@ def test_remove(tmppath):
     # Remove emptydir
     assert XRootDFS(rooturl).makedir("emptydir")
     assert XRootDFS(rooturl).remove("emptydir")
+
+
+def test_open(tmppath):
+    """Test fs.open()"""
+    # Create a file to open.
+    file_name = 'whatis'
```

Comment:
Would it be possible to just add a file in ``tests/data/`` so you don't have to write it here?